scripts/meeting_visit_count.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import datetime
+import logging
+import sys
+import requests
+from modularodm import Q
+
+from website import settings
+from website.app import init_app
+from website.models import Tag, Conference
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def main(dry_run=True):
```

Comment:
See [this comment](https://github.com/CenterForOpenScience/osf.io/pull/5128/files#r55255193)